modules/ctrl_dbus/baresipbus.c

```diff
@@ -1,8 +1,9 @@
 /*
- * Generated by gdbus-codegen 2.56.4. DO NOT EDIT.
+ * This file is generated by gdbus-codegen, do not modify it.
```

Comment:
should we consider to remove the generated file from git source control,
and generate it as part of the build process ?

something like this in module/ctrl_dbus/module.mk:

```make
module/ctrl_dbus/baresipbus.c: dbus-source.foo
	dbus-generate <param1> <params2>
```

Reply:
Yes, I think it makes sense.